lib/src/lib/dolos.ts

```diff
@@ -113,11 +129,19 @@ export class Dolos {
         nameCandidate = path.basename(paths[0]) + " & " + path.basename(paths[1]);
       }
     }
-    let ignoredFile;
-    if (ignore) {
-      ignoredFile = (await readPath(ignore)).ok();
-    }
-    return this.analyze((await files).ok(), nameCandidate, ignoredFile);
+      if (files) {
+        if (ignore) {
+          ignoredFile = (await readPath(ignore)).ok();
+        }
+        return this.analyze((await files).ok(), nameCandidate, ignoredFile);
+      }
+      else if (files_from_csv) {
+        const { nonIgnoredFiles, ignoredFile } = (await files_from_csv).ok();
+        return this.analyze(nonIgnoredFiles, nameCandidate, ignoredFile);
```

Comment:
This will ignore an `ignore` file if it was given through the CLI (by passing the `--ignore` option). I think we might try supporting that as well.

If we continue with the current approach: I think it makes sense if the previous if-block (starting at line 115) would set `files`and `ignoredFile`. This way you can drop this if/else block.